Anlab.Mvc/Views/Order/Index.cshtml

```diff
@@ -110,18 +119,25 @@
     <script type="text/javascript">
          $(function () {
              var table = $("#table").DataTable({
-                 "sorting": [[1, "desc"]],
+                 "sorting": [[5, "desc"]],
                  "columnDefs": [
                      {
-                         "type": "date", "targets": [1, 2]
-                     },
-                     {
-                         "targets": [0],
-                         "orderable": false
+                         "type": "date", "targets": [4, 5]
                      }
                  ],
                  "stateSave": true,
                  "stateDuration": 60 * 10,
+                 "createdRow": function (row, data, index) {
+                     if (data[2] == "Created") {
```

Comment:
@laholstege hmm interesting -- you could also just add it in the <tr> up under the for loop, but i guess this works too